workflows/energyplus.py

```diff
@@ -203,281 +190,306 @@ def run_energyplus(self, isIP, run_directory, file_name, args, by_api):
-            mtr_path = os.path.join(run_directory, file_name_no_ext + '.mtr')
-            eplusoutmtr_path = os.path.join(run_directory, 'eplusout.mtr')
-            if os.path.exists(mtr_path):
-                shutil.copy(mtr_path, eplusoutmtr_path)
-
-            if isIP:
-                # run the ConvertESOMTR program to create IP versions of the timestep based output files
-                if platform.system() == 'Windows':
-                    convertESOMTR_binary = os.path.join(energyplus_root_folder, 'PostProcess', 'convertESOMTRpgm',
-                                                        'convertESOMTR.exe')
-                else:
-                    convertESOMTR_binary = os.path.join(energyplus_root_folder, 'PostProcess', 'convertESOMTRpgm',
-                                                        'convertESOMTR')
-                if os.path.exists(convertESOMTR_binary):
-                    converttxt_orig_path = os.path.join(energyplus_root_folder, 'PostProcess', 'convertESOMTRpgm',
-                                                        'convert.txt')
-                    converttxt_run_path = os.path.join(run_directory, 'convert.txt')
-                    shutil.copy(converttxt_orig_path, converttxt_run_path)
-
-                    command_line_args = [convertESOMTR_binary]
-                    try:
-                        for message in self.execute_for_callback(command_line_args, run_directory):
-                            self.callback(message)
-                    except subprocess.CalledProcessError:
-                        self.callback("ConvertESOMTR FAILED")
-                        return EPLaunchWorkflowResponse1(
-                            success=False,
-                            message="ConvertESOMTR failed for file: %s!" % full_file_path,
-                            column_data={}
-                        )
-                    # copy converted IP version of ESO file to users *.eso file
-                    ipeso_path = os.path.join(run_directory, 'ip.eso')
-                    if os.path.exists(ipeso_path):
-                        shutil.copy(ipeso_path, eso_path)
-                        os.replace(ipeso_path, eplusouteso_path)
-                    # copy converted IP version of MTR file to users *.mtr file
-                    ipmtr_path = os.path.join(run_directory, 'ip.mtr')
-                    if os.path.exists(ipmtr_path):
-                        shutil.copy(ipmtr_path, mtr_path)
-                        os.replace(ipmtr_path, eplusoutmtr_path)
-                    os.remove(converttxt_run_path)
-
-            # run ReadVarsESO to convert the timestep based output files to CSV files
+        # if isIP:
+        # set up the ESO and MTR output files for either unit conversion or just ReadVarsESO
+        # *.eso back to eplusout.eso
+        eso_path = os.path.join(output_directory, file_name_no_ext + '.eso')
+        eplusouteso_path = os.path.join(output_directory, 'eplusout.eso')
```

Comment:
Had to update a number of variable assignments to use the new output directory instead of the IDF directory.  Seems to be OK, but needs more testing.